File: dotnet/GitHub/Models/DiffEntry_status.cs
using System.Runtime.Serialization;
using System;
namespace GitHubTodoDemo.GitHub.Models {
    public enum DiffEntry_status {
        [EnumMember(Value = "added")]
        Added,
        [EnumMember(Value = "removed")]
        Removed,
        [EnumMember(Value = "modified")]
        Modified,
        [EnumMember(Value = "renamed")]
        Renamed,
        [EnumMember(Value = "copied")]
        Copied,
        [EnumMember(Value = "changed")]
        Changed,
        [EnumMember(Value = "unchanged")]
        Unchanged,
    }
}

File: php/src/GitHub/Models/DiffEntry_status.php
<?php

namespace Baywet\Githubtododemo\Github\Models;

use Microsoft\Kiota\Abstractions\Enum;

class DiffEntry_status extends Enum {
    public const ADDED = 'added';
    public const REMOVED = 'removed';
    public const MODIFIED = 'modified';
    public const RENAMED = 'renamed';
    public const COPIED = 'copied';
    public const CHANGED = 'changed';
    public const UNCHANGED = 'unchanged';
}
